test/font_test.py

```diff
@@ -1,3 +1,5 @@
+# -*- coding: utf-8 -*-
+from re import T
```

Comment:
I wonder what this import was added for? I can't see this T variable used anywhere.